mscore/file.cpp

```diff
@@ -2309,10 +2315,21 @@ Score::FileError readScore(MasterScore* score, QString name, bool ignoreVersionE
       if (synti)
             score->setSynthesizerState(synti->state());
 
-      if (suffix == "mscz" || suffix == "mscx") {
+      auto read = [](MasterScore* score, const QString& name, bool ignoreVersionError, bool autosaveDirty = true)->Score::FileError {
```

Comment:
Excellent! :)

Reply:
Do you have any idea why [this](https://github.com/musescore/MuseScore/pull/5456#discussion_r405438284) happens?